src/Dialogue.js

```diff
@@ -15,6 +15,15 @@ export default class Dialogue {
     this.completeText = props.completeText;
     this.completeHelp = props.completeHelp;
     this.completeDrops = props.completeDrops || {};
+    this.progression = props.progression;
+
+    // This gets the first field in the object.
+    // Because they're ordered.
+    // Because javascript says fuck logic.
+    for (var text in this.conversation) {
+      this.activeText = this.conversation[text];
+      break;
+    }
```

Comment:
I think this would be cleaner if you just initialize `progress` (currently the first property listed under dialog) with the key of wherever the convo starts. Then you would only have to do `this.activeText = this.conversation[this.progress]`

While JS normally does order the keys you're not really supposed to rely on that - different engines do it differently